extensions/amp-bind/0.1/test/test-bind-carousel.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../../../amp-carousel/0.1/amp-carousel';
+import {installBindForTesting} from '../bind-impl';
+import {toggleExperiment} from '../../../../src/experiments';
+import {chunkInstanceForTesting} from '../../../../src/chunk';
+import {createIframePromise} from '../../../../testing/iframe';
+import {bindForDoc} from '../../../../src/bind';
+
+describe('test-scrollable-carousel', () => {
+  let iframe;
+  let slideNum;
+  let carousel;
+  let bind;
+
+  beforeEach(() => {
+    return createIframePromise().then(i => {
```

Comment:
Check out how `createFixtureIframe` is used in other integration tests. 

They live in `/test/integration/` or in individual extension folders (searching for "test integration" in Sublime's open resource should suffice).